corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -305,7 +309,10 @@ def split_files_and_refactor(self, app_name, file_path, bootstrap3_lines, bootst
             self.write_response("ok, canceling split and rolling back changes...")
             return
 
-        has_no_existing_changes = self.has_no_existing_changes()
+        has_no_existing_changes = has_no_pending_git_changes()
+        if not has_no_existing_changes:
```

Comment:
Nitpick: the double negative makes this harder to read. A `has_pending_git_changes` function would be a little easier.

Reply:
updated